openhands/app_server/app_conversation/sql_app_conversation_info_service.py

```diff
@@ -523,7 +513,7 @@ def _to_info(
 
         return AppConversationInfo(
             id=UUID(stored.conversation_id),
-            created_by_user_id=stored.user_id if stored.user_id else None,
+            created_by_user_id=None,  # User ID is now stored in ConversationMetadataSaas
```

Comment:
```suggestion
            created_by_user_id=None,  # User ID is now stored in ConversationMetadata
```

i believe this is the local metadata ?

Reply:
The idea is to remove user_id from the OSS conversation metadata as it's not applicable.